settings.gradle.kts

```diff
@@ -34,6 +34,7 @@ include(
     ":opentelemetry-kotlin-exporters-core",
     ":opentelemetry-kotlin-exporters-otlp",
     ":opentelemetry-kotlin-exporters-protobuf-java",
```

Comment:
This module can be deleted in favour of `opentelemetry-kotlin-exporters-protobuf`